decidim-core/app/helpers/decidim/sanitize_helper.rb

```diff
@@ -23,6 +23,13 @@ def decidim_sanitize(html, options = {})
       end
     end
 
+    def sanitize_content_for_comment(text, options = {})
```

Comment:
I don't like having the `sanitize_content_for_comment` inside of `decidim-core`. Can you move this to `decidim-comments/app/cells/decidim/comments/comment_cell.rb` 🙏🏽 ?